diagnostics/ocn_surf_flux_diag/groupby_variables.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+import xarray as xr
+import warnings
+from scipy import stats
+
+
+
+
+def bin_2d(ds,bin1_var,bin2_var,target_var,bin1=10,bin2=10,stTconfint=0.99,bin1_range=None,bin2_range=None):
```

Comment:
Can `xhistogram` achieve this same functionality?  https://xhistogram.readthedocs.io/en/latest/

Reply:
@jkrasting Thank you for the suggestion! However, after checking the function, I think it is missing the statistical analysis that the POD needs during the binning.